common/source/docs/common-marvelmind.rst

```diff
@@ -22,29 +22,38 @@ Required Hardware
+- Set the stationary beacon with the lowest address as the origin in your dashboard.
+- The height of every stationary beacon, from the ground, must be set on the dashboard.
+- The hedge should be set to output Marvelmind Protocol at baud rate of 115200.
+
 Connecting to a flight controller
 =================================
 
-- ensure the mavelmind is firmware version 5.77 (or higher)
-- ensure that Copter-3.6 (or higher) or Rover-3.3 is loaded onto the flight controller and connect with a Ground Station (i.e. Mission Planner).
-- enable the :ref:`EKF3 as described here <common-apm-navigation-extended-kalman-filter-overview>`
-- set :ref:`EK3_ALT_SOURCE <EK3_ALT_SOURCE>` to 3 (means using Marvelmind for altitude)
-- set :ref:`BCN_TYPE <BCN_TYPE>` to 2 (means using Marvelmind system)
-- set :ref:`BCN_LATITUDE <BCN_LATITUDE>`, :ref:`BCN_LONGITUDE <BCN_LONGITUDE>` and :ref:`BCN_ALT <BCN_ALT>` to match your actual location.  Getting these values exactly correct is not particularly important although getting it close is required in order for the compass's declination to be correctly looked up from the small database held within ArduPilot.
-- set :ref:`BCN_ORIENT_YAW <BCN_ORIENT_YAW>` to the heading from the origin anchor to the 2nd anchor.  One way to capture this value is to stand at the origin holding the vehicle so that it's nose points towards the second beacon.  Read the vehicle's heading from the HUD and enter this value into :ref:`BCN_ORIENT_YAW <BCN_ORIENT_YAW>`
-- set :ref:`GPS_TYPE <GPS_TYPE>` to 0 to disable the GPS
-- set :ref:`ARMING_CHECK <ARMING_CHECK>` to -9 to disable the GPS arming check
-- set :ref:`SERIAL1_BAUD <SERIAL1_BAUD>` to 115 to set telemetry1's baud rate to 115200 (if using Telem2 set :ref:`SERIAL2_BAUD <SERIAL2_BAUD>` instead)
-- set :ref:`SERIAL1_PROTOCOL <SERIAL1_PROTOCOL>` to 13 to enable receiving data from the marvelmind (If using Telem2 set :ref:`SERIAL2_PROTOCOL <SERIAL2_PROTOCOL>` instead)
-- set :ref:`BRD_SER1_RTSCTS <BRD_SER1_RTSCTS>` to 0 to ensure telem1 does not use flow control (If using Telem2 set :ref:`BRD_SER2_RTSCTS <BRD_SER2_RTSCTS>` instead, not required if using Serial4/5)
+- Ensure that Copter-3.6 (or higher) or Rover-3.3 is loaded onto the flight controller and connect with a Ground Station (i.e. Mission Planner).
+- Enable the :ref:`EKF3 as described here <common-apm-navigation-extended-kalman-filter-overview>`
+- Set :ref:`EK3_ALT_SOURCE <EK3_ALT_SOURCE>` to 3 (means using Marvelmind for altitude)
+- Set :ref:`EK3_GPS_TYPE <EK3_GPS_TYPE>` to 3 (means Inhibit GPS use)
+- Set :ref:`BCN_TYPE <BCN_TYPE>` to 2 (means using Marvelmind system)
+- Set :ref:`BCN_LATITUDE <BCN_LATITUDE>`, :ref:`BCN_LONGITUDE <BCN_LONGITUDE>` and :ref:`BCN_ALT <BCN_ALT>` to match your actual location.  Getting these values exactly correct is not particularly important although getting it close is required in order for the compass's declination to be correctly looked up from the small database held within ArduPilot.
```

Comment:
there may be an issue with the declinaison as we correctly set the position but declinaison stay at 0 ... I am not sure if that part works

Reply:
Ah okay. This was written by @rmackay9 for Pozyx also. I'm guessing it somehow works?

I would really love to remove dependency of magnetic compass for GPS denied environments, because indoors it is really so bad